src/languages/ldif.js

```diff
@@ -9,7 +9,10 @@ function(hljs) {
       {
         className: 'attribute',
         begin: '^dn', end: ': ', excludeEnd: true,
-        starts: {end: '$', relevance: 0},
+        relevance: 10
+      },
+      {
+        begin:'(cn|CN|ou|OU|dc|DC)=',
         relevance: 10
```

Comment:
Are these not attributes also?  Did we not need that starts/ends $ match?

Reply:
Oh now that's a good idea, would have been useful to see it with the tests so I had some idea what your'e trying to accomplish. :)

Reply:
Sorry - it's purely an effort to get the autodetect to not match LDIF as YAML. I figured this was closer to the original, and by adding a lot of extra `dn` entries to the LDIF default test, it seems to do the trick.

Reply:
Yeah, sorry.  The autodetect stuff does make contributing a real pain.  I'm trying to think of some way to improve it.

Reply:
So we’re end and begin meant to swap here then?

Reply:
Nope that was a stuff-up. I've just pushed a fix.